README.md

```diff
@@ -1668,6 +1668,10 @@ This parameter is only used if the [`verify_config`][] parameter's value is 'tru
 
 Specifies whether to validate the configuration file before notifying the Apache service. Valid options: Boolean. Default: `true`.
 
+##### `default_ext`
+
+Specifies what extension to append on end of custom config file. Set to '' to keep original extension. Default: `.conf`.
```

Comment:
It would require that your original source file be named with no extension, which seems weird. You might not know if "rewrite" is a conf or common extension.